src/scrabble.js

```diff
@@ -1,5 +1,47 @@
-function scrabble() {
+const scoreDict = {
+  score1: ['A', 'E', 'I', 'O', 'U', 'L', 'N', 'R', 'S', 'T'],
+  score2: ['D', 'G'],
+  score3: ['B', 'C', 'M', 'P'],
+  score4: ['F', 'H', 'V', 'W', 'Y'],
+  score5: ['K'],
+  score8: ['J', 'X'],
+  score10: ['Q', 'Z']
+}
+
+function scrabble(str) {
+  if (!validateString(str)) return 0
+
+  let score = 0
+  let doubleValue = false
+  let tripleValue = false
+
+  for (const char of str.toUpperCase().split('')) {
+    if (char === '{') doubleValue = true
+    else if (char === '[') tripleValue = true
+    else if (char === '}') doubleValue = false
+    else if (char === ']') tripleValue = false
 
+    const returnScore = () => {
+      for (const key of Object.keys(scoreDict)) {
+        if (scoreDict[key].includes(char)) {
+          let num = 1
+          num = doubleValue ? num * 2 : tripleValue ? num * 3 : num
```

Comment:
Like the use of ? instead of if statements

Reply:
😅 always nice to make it in fewer lines

👀 as long as it doesn't break HAHAH